packages/cli/src/databases/postgresdb/migrations/1636626154934-CreateUserManagement.ts

```diff
@@ -0,0 +1,146 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { v4 as uuid } from 'uuid';
+import config = require('../../../../config');
+import { loadSurveyFromDisk } from '../../utils/migrationHelpers';
+
+export class CreateUserManagement1636626154934 implements MigrationInterface {
+	name = 'CreateUserManagement1636626154934';
+
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		const tablePrefix = config.get('database.tablePrefix');
+		await queryRunner.query(
+			'CREATE TABLE "' + tablePrefix + 'role" (' +
+				'"id" serial NOT NULL,' +
+				'PRIMARY KEY ("id"),' +
```

Comment:
previously we used `CONSTRAINT "PK_${tablePrefixPure}a60448a90e51a114e95e2a125b3" PRIMARY KEY ("column1", "column2")`

I think we then should stick to the format